programs/vote/src/vote_state/mod.rs

```diff
@@ -32,10 +32,6 @@ pub const INITIAL_LOCKOUT: usize = 2;
 //  smaller numbers makes
 pub const MAX_EPOCH_CREDITS_HISTORY: usize = 64;
 
-// Frequency of timestamp Votes. In v0.22.0, this is approximately 30min with cluster clock
-// defaults, intended to limit block time drift to < 1hr
-pub const TIMESTAMP_SLOT_INTERVAL: u64 = 4500;
```

Comment:
why not just speed this up by 10x, instead of every vote?

Reply:
Why not every vote though?  It's not a lot of extra data relative to the size of a vote transaction

Reply:
@aeyakovenko - make sense?